examples/with-lambda-scheduling/main.tf

```diff
@@ -0,0 +1,52 @@
+####################
+# Actual Eventbridge
+####################
+module "eventbridge" {
+  source = "../../"
+
+  # Schedules can only be created on default bus
+  create_bus = false
+
+  # Fire every five minutes
+  rules = {
```

Comment:
Let's merge these `rules`, `targets` with the rest of the things inside of the `complete` example. I think there is no need to have this as a separate example.

PS: Try to run an example yourself as a confirmation that it works as you expect.

Reply:
I added in the complete example as requested, but I also left a specific example for lambdas following the ECS pattern. Wouldn't it be interesting to keep a separate example for lambdas as in the other scenarios?

PS.: I ran the specific example of lambdas on my local machine, I needed to create a folder with a very basic code to define the code that would go to the lambda and also changed the variables of the modules that were being retrieved. Outra coisa importante é que para executar esse exemplo é necessário o Python 3.